### DISCUSSION TITLE
Opening scripts uses previous db connection

### DISCUSSION THREAD

**Discussion by Patrick12589** [dbeaver/dbeaver#11230] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
So I deployed 5 scripts to out UAT/PAT environment
When I try & to deploy them to live it remembers the old connection and even though I close the connection & open a new one it keeps defaulting back to this old/disconnected session - how do I stop this?
I'm dragging the .sql scripts into DBeaver
I feel this is quite dangerous for me - since I could deploy the scripts very easily in the wrong environment based on where I ran them previously - better to only use current connections

**Comment by Patrick12589** [comment, marked as answer]
is the answer auto-sync connection with navigator? - testing what this does - can't find any documentation on what this does
but it seems to fix the issue

**Comment by uslss** [comment]
Auto-sync connection with navigator option automatically sets database that is chosen in DB navigator as a default for the current connection in SQL editor.
![image](https://user-images.githubusercontent.com/31996417/106912261-e93c1300-6713-11eb-9a38-1ba2aea6a447.png)
